.github/workflows/ci-build.yaml

```diff
@@ -258,6 +282,13 @@ jobs:
           path: /tmp
       - name: Load argoexec image
         run: docker load < /tmp/argoexec_image.tar
+      - name: Download argocli image
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
```

Comment:
is there a way to parallelize this so that we download both images simultaneously?

Reply:
Yes, I believe so. Again, optimising for understand-ability and trying to fix CI.